python/python/glide/async_commands/standalone_commands.py

```diff
@@ -329,6 +330,70 @@ async def sort(
         result = await self._execute_command(RequestType.Sort, args)
         return cast(List[Optional[str]], result)
 
+    async def sort_ro(
+        self,
+        key: str,
+        by_pattern: Optional[str] = None,
+        limit: Optional[Limit] = None,
+        get_patterns: Optional[List[str]] = None,
+        order: Optional[OrderBy] = None,
+        alpha: Optional[bool] = None,
+    ) -> List[Optional[str]]:
+        """
+        Sorts the elements in the list, set, or sorted set at `key` and returns the result.
+        The `sort` command can be used to sort elements based on different criteria and apply transformations on sorted elements.
+        This command is routed depending on the client's `ReadFrom` strategy.
```

Comment:
I think it is also not applicable in transaction implementation, because the entire transaction is routed, not a particular command.
```suggestion
```